supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/Run.scala

```diff
@@ -92,11 +92,16 @@ case class Run(runId: String, genomicsInterface: Genomics) {
   def status(): RunStatus = {
     val op = genomicsInterface.operations().get(runId).execute
     if (op.getDone) {
-      // If there's an error, generate a Failed status. Otherwise, we were successful!
       val eventList = getEventList(op)
+      val ceInfo = op.getMetadata.get ("runtimeMetadata").asInstanceOf[GArrayMap[String,Object]].get("computeEngine").asInstanceOf[GArrayMap[String, String]]
+      val machineType = ceInfo.getOrDefault("machineType","unknown")
+      val instanceName = ceInfo.getOrDefault("instanceName","unknown")
+      val zone = ceInfo.getOrDefault("zone","unknown")
```

Comment:
I feel like it would be more intellectually honest to just return these as `Option`s